frame/treasury/src/lib.rs

```diff
@@ -113,6 +114,9 @@ type NegativeImbalanceOf<T> = <<T as Trait>::Currency as Currency<<T as frame_sy
 /// The treasury's module id, used for deriving its sovereign account ID.
 const MODULE_ID: ModuleId = ModuleId(*b"py/trsry");
 
+/// The maximum size of tippers, this is used to compute weight, the exceeding will be refund.
+pub const MAX_TIPPERS_COUNT: u64 = 1000;
```

Comment:
I think we should generally make these kinds of numbers a configurable trait. Any reason you see not to?

Reply:
actually I thought there was as much member as validator but actually no.
in Kusama the value is 13 so negligible, I'll change to a configurable max and no refund.

Reply:
I implemented an upper bound in the associated type directly.